core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1521,7 +1521,7 @@ private[spark] class DAGScheduler(
             markStageAsFinished(failedStage, errorMessage = Some(failureMessage),
               willRetry = !shouldAbortStage)
           } else {
-            logDebug(s"Received fetch failure from $task, but its from $failedStage which is no " +
+            logDebug(s"Received fetch failure from $task, but it's from $failedStage which is no " +
               s"longer running")
```

Comment:
Can you drop `s` in the head, too?

Reply:
@maropu  Do you mean the `s` of `s"Received..."`? It's needed, since variables `$task` and `$failedStage` are used in the message string.

Reply:
s"longer running" <- here

Reply:
done.